aw_client/client.py

```diff
@@ -149,18 +154,33 @@ def get_eventcount(self, bucket_id: str, limit: int=100, start: datetime=None, e
         response = self._get(endpoint, params=params)
         return int(response.text)
 
-    def heartbeat(self, bucket_id: str, event: Event, pulsetime: float, queued: bool=False) -> Optional[Event]:
+    def heartbeat(self, bucket_id: str, event: Event, pulsetime: float, queued: bool=False):
         """ This endpoint can use the failed requests retry queue.
             This makes the request itself non-blocking and therefore
             the function will in that case always returns None. """
 
+        from aw_transform.heartbeats import heartbeat_merge
         endpoint = "buckets/{}/heartbeat?pulsetime={}".format(bucket_id, pulsetime)
-        data = event.to_json_dict()
```

Comment:
I don't think premerging of heartbeats should be done at all if `queued=False`, would make the code clearer as well I think.

Reply:
True.

Something unrelated to this PR though, queue should probably be on by default. In most cases the watcher should be dumb and only send data and not really care about the response.